### DISCUSSION TITLE
[Feature] Make the shared album asset count visible to all users

### DISCUSSION THREAD

**Discussion by jon-f-novabackup** [immich-app/immich#2301] [Feature Request]
### The feature

Display the total items (assets) count by each album that is listed for shared albums in the "Sharing" section, and at the top when viewing a shared album (after the album loads and is displayed), for instance near the date just under the album name, so that not just the owner of the shared album can see that stat easily. Note: In 1.54.0 only the owner of the shared album is able to see this stat easily but not the other shared members, and the app that shall not be named also has this for all users to easily see. Screenshots to describe this are below:

Albums section in google (web version), where the user that is currently logged in to google website is simply a member of the shared album but is not the owner, the total items count is below each album name in '/albums':
![image](https://user-images.githubusercontent.com/50121594/233485398-cdbd1cc1-4136-47e1-9f48-702eaa06753a.png)

Sharing section in google (mobile app version, Android), where the user that is currently logged in to google app is simply a member of the shared album but is not the owner, does not display the assets count below each album name (the web version does do this, see the above noted item screenshot), but a feature like that in Immich would certainly be good to have in the mobile app I think:
![image](https://user-images.githubusercontent.com/50121594/233493377-513442fe-a67d-47e7-bebd-6f5f364ae317.png)

Sharing section in Immich (1.54.0) website, the listing of shared albums does not display the assets count (just who the owner is):
![image](https://user-images.githubusercontent.com/50121594/233486866-d7c3833a-f3e8-4ac5-81a6-4f66ab613490.png)

Sharing section in Immich (1.54.0) website, after viewing / loading one of the above shared albums, also does not display the image count anywhere here, unless possibly when go to add a photo; I think that this would be a nice to have feature to display the assets count somewhere at the top in an easy to see place in this dialog, optimally under the album name near the date at the top, Note: google doesn't do this either when viewing an album in app or web view:
![image](https://user-images.githubusercontent.com/50121594/233487809-d86c454f-765c-4bc0-8677-85115e64f215.png)

Sharing section in Immich (1.54 build.77) Android mobile app, here I'm logged in as the owner (the "John Doe" user) of the shared album this time, and the listing of shared albums does not show a total items count (just who the owner is, in this case as "Owned"):
![image](https://user-images.githubusercontent.com/50121594/233487174-b8f7a845-042b-4b53-a0e2-5299738cccd8.png)

Sharing section in Immich (1.54 build.77) Android mobile app, here I'm logged in as just a member of the shared album but not the owner this time, and the listing of shared albums does not show a total items count (just who the owner is, "John Doe"):
![image](https://user-images.githubusercontent.com/50121594/233494352-906604be-d498-40d0-88b8-19d14f4ee954.png)

Sharing section in Immich (1.54 build.77) Android mobile app, after viewing / loading one of the above shared albums, does not show a total items count (this is a nice to have somewhere easy to read on this dialog optimally near the date, google doesn't do it either):
![image](https://user-images.githubusercontent.com/50121594/233489448-2535b364-29d0-47bf-9dd9-e7ba4c3b836a.png)

And here is the same feature request described in a more descriptive way, that could be useful for some:
It would be nice if the "Sharing" section of the website and mobile app could show the total images count in the shared albums list (in any section that deals with that), as currently if you are not the owner of an album it only shows "you are the owner" text or tells you who the owner is there, but no total image count in the listing of shared albums. Users that have access to the shared album(s) who are not the owner don't see any albums in the "Albums" section to be able to see the total image count. In addition to that, the website and the mobile app should show a count of the images at the top after it loads, just under the album name. The only place that the total image items count is seen is some dialogs to do with adding photos to a shared album, and inside the [Albums] section, but that section currently only displays albums that you are the owner of, and the [Library] section in the mobile app again only displays albums that you are the owner of (so the count is not seen there).

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by faupau03** [comment]
Hey, thanks for your feature request! I implemented this feature on the web side in my pull request #2311.

**Reply by jon-f-novabackup** [reply]
@faupau03 thanks for getting that feature implemented on the web side so quickly. That is exactly what I was looking for like how your example screenshots show for both the shared albums listing and when actually viewing a shared album, to display the total asset count in both areas! I hope it can be implemented the same way in mobile soon.
